.changeset/tasty-rabbits-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"providence-analytics": feat
```

Comment:
@tlouisse be careful when you create these manually. This crashes changesets. Please run `yarn changeset` and use the menu, then edit the summary later if needed.